index.html

```diff
@@ -9,17 +9,24 @@
   </head>
   <body>
     <header class="sec">
-      <nav class="sec-cont">
-        <ul>
-          <li><a href="#">Filmer</a></li>
-          <li><a href="#">Salonger</a></li>
-          <li><a href="#">Restaurang</a></li>
-          <li><a href="#">Events</a></li>
-          <li><a href="#">Om oss</a></li>
-        </ul>
-      </nav>
+      <div class="header-content-wrap">
+        <img src="static/images/logo.png" alt="logo" />
+
+        <nav class="header-nav">
+          <button class="header-nav-toggle"></button>
+          <menu class="header-menu">
+            <li>
+              <a class="header-menu-item active" href="/index.html">Hem</a>
+            </li>
+            <li><a class="header-menu-item" href="#">Om oss</a></li>
+            <li><a class="header-menu-item" href="#">Biljettinfo</a></li>
+            <li><a class="header-menu-item" href="#">Presentkort</a></li>
+          </menu>
+        </nav>
+      </div>
     </header>
     <main></main>
     <footer></footer>
+    <script type="module" src="/src/JS/main.js"></script>
```

Comment:
Maybe, we could have this in head with "defer" instead. It could be nice to have it together with the rest of the content in head. With defer javascript loads after html is finished.

<script type="module" src="/src/JS/main.js" defer></script>

Reply:
Sure. I read something about when "type=module" you dont need defer in there. That its ok to put in head without it. I can test that.